### ISSUE TITLE
Consider Sphinx and ReadTheDocs for generating/hosting documentation

### ISSUE THREAD

**Issue by willfurnass** [N8-CIR-Bede/documentation#2]
Would allow RSEs/RIEs to contribute fixes/new content via PRs and would reduce maintenance burden on core admins.

**Comment by MarionBWeinzierl** [comment]
@bodgerer , were there any plans about where to move the docs from this wiki?

**Comment by bodgerer** [comment]
The plan was to move the documentation to the N8CIR website. I think Nikki has largely done this, and is waiting on the advice for applying for projects.

**Comment by bodgerer** [comment]
The original idea was that RSEs would be given permissions to edit the pages on the website. Departing from this would need to be discussed with Niki and Gillian.

**Comment by MarionBWeinzierl** [comment]
I am checking with Niki what the plan is

**Comment by bodgerer** [comment]
I had a chat with Niki and Gillian.

They're happy to move the technical documentation off the main website. We 
can do this but keep the n8cir url as the publicised entry point, which 
tells you how to apply and where the rest of the documentation is.

If people have a strong feeling about where the documentation should live, 
that's great. But as whatever it is will be new to me, they'll need to 
help make it happen :)

Mark

**Comment by MarionBWeinzierl** [comment]
Niki suggested discussing in tomorrow's meeting where to host the docs

**Comment by jonathanfrawley** [comment]
Okay, I have transferred over what @bodgerer created in the Wiki section into the ReadTheDocs format. I have also added some info on Profiling and Training from the Sheffield GPU Hack website. If people could review this and make sure they are happy with it that would be great.

The next step is to create a ReadTheDocs account and link it to this repo. I think that should be it. I guess this should probably wait until the repo is moved to the new Bede GitHub organisation. @MarionBWeinzierl I think you should be the owner of this ReadTheDocs account as well.

**Comment by bodgerer** [comment]
Thanks to @jonathanfrawley for doing some fine work, but readthedocs is showing an ad alongside our documentation.

**Comment by MarionBWeinzierl** [comment]
I changed the settings so that no paid ads are shown, but it seems that we cannot completely avoid ads?

![image](https://user-images.githubusercontent.com/56403724/99404007-09adbb80-28e3-11eb-89f3-b13ce3f3cd9c.png)

**Comment by MarionBWeinzierl** [comment]
I am closing this issues, as we have considered the alternative hosting, made a decision, and moved the documentation over. Please open a new issue for any further documentation topics.
